html2text/__init__.py

```diff
@@ -437,6 +438,17 @@ def no_preceding_space(self):
                     self.abbr_title = None
                 self.abbr_data = ''
 
+        if tag == "q":
+            # Different languages use different quotation marks.
+            # It should be easy to change the punctuation characters.
```

Comment:
When it says 

> Different languages use different quotation marks.

What do you mean by different "quotation marks" ?

And how about 

> It should be easy to change the punctuation characters.

How can be **the punctuation characters** can be changed

Reply:
Ah! So in English the open and close quotation marks are symmetrical: `"..."`. Other languages use different symbols, like `«...»`, or `「...」`, or `《...》`. I assumed that users would just change the open and close characters in the code:

            if not self.quote:
                # Open quote
                self.o('«')
            else:
                # Close quote
                self.o('»')

As I'm writing this, I realize I didn't actually test using characters that require Unicode -- there may be an issue with Unicode output. I'll test Unicode output and report back.

Reply:
I've discovered how the CLI, and options in general, are set up. I'm updating the code now and will push the changes as soon as unit tests pass.